packages/e2e/features/ui/components/authenticator/modal.feature

```diff
@@ -1,15 +1,15 @@
-Feature: withAuthenticator
+Feature: Modal
 
-  Higher-Order Component gates an application with the Authenticator
+  Modal variation overlays an application with the Authenticator
 
   Background:
     Given I'm running the example "/ui/components/authenticator/withAuthenticator"
 
-  @react
+  @skip @angular @react @vue
   Scenario: Application is wrapped with Authenticator
     Then I see "Sign in"
 
-  @react
+  @skip @angular @react @vue
   Scenario: Application renders when signed in
     When I type my "username" with status "CONFIRMED"
     And I type my password
```

Comment:
In the future, I think we can enhance this test to test for *actual* model behavior.

Reply:
In React, there's some default text in the app that we could do:

```
And I don't see "Example App"
```

https://github.com/aws-amplify/amplify-ui/blob/038075d4e47a5e213f44ab7798821b9c483be131/examples/next/pages/_document.page.tsx

Would that work?

Reply:
Actually, I'd rather have percy.io & visual regression tests cover this.

https://docs.cypress.io/guides/core-concepts/interacting-with-elements#Visibility isn't accurate enough to detect _it's not actually visible_.